tests/unit/test-libm.c

```diff
@@ -39,7 +39,14 @@ check_double (const char *expr, double computed, double expected)
   uint64_t *computed_bits_p = (uint64_t *) &computed;
   uint64_t *expected_bits_p = (uint64_t *) &expected;
 
-  printf ("%s = 0x%lx [expected=0x%lx] ", expr, *computed_bits_p, *expected_bits_p);
+  uint32_t computed_hi = (uint32_t) (*computed_bits_p >> 32);
+  uint32_t computed_lo = (uint32_t) *computed_bits_p;
+  uint32_t expected_hi = (uint32_t) (*expected_bits_p >> 32);
+  uint32_t expected_lo = (uint32_t) *expected_bits_p;
```

Comment:
There is a better way to do this with unions. Define a union with a double and an uint32_t [2] array, and do the conversion with it.